INSTALL.txt

```diff
@@ -35,92 +34,79 @@
     or:
         $ perl Configure.pl --gen-nqp --backends=jvm  # needs JDK 1.8 installed
     then:
-        $ make
+        $ make install
```

Comment:
Not `make; make install`, but just `make install`? I am not sure what works for rakudo, just seeing the former often.

Reply:
I use `make install` always when installation is the final target.